### PR TITLE
Add full size logo to readme as header

### LINKED ISSUE #1833: Add full size logo to readme as header
Use build/logo.svg upsize it so the width fills the header

### PR DESCRIPTION
**Pull request by mnyamunda-scottlogic**
[Base Branch]: #
# Base Branch
dev

Closes #1833 

[Why]: #
 # Why: 
This allows users to instantly recognise the product by it's logo

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,6 @@
-# tgstation-server
+<div style= "text-align: center;">
+<img src =https://raw.githubusercontent.com/tgstation/tgstation-server/dev/build/logo.svg>
+</div>
```
Comment (Cyberboss) [inline README.md:3]: Try
```
<p align="center">
  <img src =./build/logo.svg>
</p>
```

**Review by Cyberboss — CHANGES_REQUESTED** [review]

**Comment by Cyberboss** [comment]
R.e. your PR description I made some edits:

- Changed the base branch to dev, because you branched off dev and committing to master would have brought other dev changes over too soon.
- Removed the release notes as it's not a user facing change.
- Adjusted the issue reference so merging this PR will close it.

File: README.md (hunk 1 above)
Comment (mnyamunda-scottlogic) [reply README.md:3]: Will do! This makes sense as the file is already in the codebase!

**Comment by mnyamunda-scottlogic** [comment]
> R.e. your PR description I made some edits:
> 
> * Changed the base branch to dev, because you branched off dev and committing to master would have brought other dev changes over too soon.
> * Removed the release notes as it's not a user facing change.
> * Adjusted the issue reference so merging this PR will close it.

Appreciate it, I had closed my previous PR due to those other dev changes 😆

File: README.md (hunk 1 above)
Comment (mnyamunda-scottlogic) [reply README.md]: Hi Cyberboss, this change has been committed and pushed :)

File: README.md [hunk 2]
```diff
@@ -1,4 +1,6 @@
-# tgstation-server
```
Comment (Cyberboss) [inline README.md:1]: This line shouldn't have been deleted. Also there are now several unrelated changes present which need to be undone.

**Review by Cyberboss — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (mnyamunda-scottlogic) [reply README.md:1]: All good, I think those new unrelated changes came from merging the dev branch into this one. Will revert those :)

**Review by Cyberboss — CHANGES_REQUESTED** [review]
Still a lot of unrelated changes: https://github.com/tgstation/tgstation-server/pull/1841/files

File: README.md [hunk 3]
```diff
@@ -661,4 +665,4 @@ Feel free to ask for help [on the discussions page](https://github.com/tgstation
 * The /tg/station 13 icon is licensed under [Creative Commons 3.0 BY-SA](http://creativecommons.org/licenses/by-sa/3.0/).
 * The remainder of the project is licensed under [GNU AGPL v3](http://www.gnu.org/licenses/agpl-3.0.html)
 
-See the files in the `/src/DMAPI` tree for the MIT license
+See the files in the `/src/DMAPI` tree for the MIT license
```
Comment (Cyberboss) [inline README.md:668]: Last one is a missing new line.

Comment (mnyamunda-scottlogic) [reply README.md:668]: Does this mean that line 669 should be blank?

Comment (Cyberboss) [reply README.md:668]: Yes

Comment (mnyamunda-scottlogic) [reply README.md]: All done! Thank you for your patience and responsiveness Cyberboss :)
